code/controllers/stonedmc/subsystem/ticker.dm

```diff
@@ -55,16 +55,19 @@ SUBSYSTEM_DEF(ticker)
 	switch(current_state)
 		if(GAME_STATE_STARTUP)
 			if(Master.initializations_finished_with_no_players_logged_in)
+				start_at = world.time + ((CONFIG_GET(number/lobby_countdown) * 10) * 2)
+			else
 				start_at = world.time + (CONFIG_GET(number/lobby_countdown) * 10)
```

Comment:
Keeping this line and removing the three lines before, will set the counter when the first client connect to the server.
Also, if you do that, you can remove the `start_at = world.time + (CONFIG_GET(number/lobby_countdown) * 10)` set during `initialize`.